Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -602,9 +617,13 @@ static Color activeButtonTextColor()
             }
         };
 
-        if (auto selector = selectCocoaColor()) {
-            if (auto color = wtfObjCMsgSend<NSColor *>([NSColor class], selector))
-                return semanticColorFromNSColor(color);
+        if (auto [selector, makeOpaque] = selectCocoaColor(); selector) {
+            if (auto nsColor = wtfObjCMsgSend<NSColor *>([NSColor class], selector)) {
+                auto color = semanticColorFromNSColor(nsColor);
+                if (makeOpaque)
+                    color = blendSourceOver(localAppearance.usingDarkAppearance() ? Color::black : Color::white, color).semanticColor();
```

Comment:
You should blend over the system color named Canvas if you can.

Reply:
Thanks for the tip, I'll give it a try.

Note that I've copied what was on the iOS side: https://searchfox.org/wubkat/rev/1dfbd6f7279ea1201b5afb9719b4163c8fab1566/Source/WebCore/rendering/RenderThemeIOS.mm#1136
Should I try to change that one to use the "Canvas" color as well? (Maybe in a separate bug?)